x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -79,7 +81,16 @@ export function ErrorDistribution({ distribution, title }: Props) {
       <Histogram
         tooltipHeader={tooltipHeader}
         verticalLineHover={(bucket: FormattedBucket) => bucket.x}
-        xType="time"
+        xType="time-utc"
+        formatX={(value: any) => {
```

Comment:
is there a better type available?

Reply:
Not yet, https://github.com/uber/react-vis/issues/636

Reply:
is it not a `Date` object?

Reply:
Found it, it's indeed a data. `tickFormat(): ((d: Date) => string);`